examples/sample-app/cleanup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 echo "Killing openshift all-in-one server ..."
-killall openshift
+pkill --exact openshift
```

Comment:
Thanks, looks like you're right and Mac doesn't have a long version. I've corrected it.